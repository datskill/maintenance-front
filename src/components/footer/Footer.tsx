import React from 'react';
import './footer.css'
import { Grid } from '@material-ui/core';

export default function Footer() {
    return (
        <footer className="fixed-bottom">
            <Grid
                container
                direction="row"
                justify="flex-start"
                alignItems="center"
                className="empty-div"
            >
                <Grid item md={4} xs={6} className="styled-copyright">
                    OpenData Paris Copyright 2021
                </Grid>
                <Grid item md={4} xs={6} className="styled-copyright">
                    EPSI MSPR Maintenance Applicative
                </Grid>
            </Grid>
        </footer >
    );
}