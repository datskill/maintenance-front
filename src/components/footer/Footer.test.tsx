import Footer from "./Footer";
import * as React from "react";
import * as renderer from "react-test-renderer";

it("Footer renders correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
