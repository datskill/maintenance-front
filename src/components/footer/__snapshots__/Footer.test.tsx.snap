// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="fixed-bottom"
>
  <div
    className="MuiGrid-root empty-div MuiGrid-container MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root styled-copyright MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-md-5"
    >
      OpenData Paris Copyright 2021©
    </div>
    <div
      className="MuiGrid-root styled-description MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-md-3"
    >
      EPSI MSPR Maintenance Applicative
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-md-4"
    >
      <p
        className="MuiTypography-root styled-maddewithlove MuiTypography-body1 MuiTypography-alignRight"
      >
        Made with ❤ in EPSI Montpellier
      </p>
    </div>
  </div>
</footer>
`;
